tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -2315,15 +2151,14 @@ def test_kpods_fail_node_nw_down(self):
     @pytest.mark.lc
     @pytest.mark.skip(reason="Not supported in RGW yet")
```

Comment:
we can remove skip marker as this will be supported in PI8

Reply:
Will take care of this in next PR with optimization and removal of skip marker and pytest warning disable comments